src/main/java/org/datadog/jmxfetch/util/ServiceCheckHelper.java

```diff
@@ -0,0 +1,12 @@
+package org.datadog.jmxfetch.util;
+
+import org.apache.commons.lang.StringUtils;
+
+public class ServiceCheckHelper {
+    /** Formats the service check prefix. */
+    public static String formatServiceCheckPrefix(String fullname) {
```

Comment:
is this function tested anywhere? (if not, we should add a test for it to improve test coverage a bit)

Reply:
Sure, added a test for it.